roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/settings/AppSettingsComponent.kt

```diff
@@ -3,26 +3,44 @@ package com.github.takahirom.roborazzi.idea.settings
 import com.intellij.ui.components.JBLabel
 import com.intellij.ui.components.JBTextField
 import com.intellij.util.ui.FormBuilder
+import javax.swing.Box
+import javax.swing.BoxLayout
 import javax.swing.JComponent
+import javax.swing.JLabel
 import javax.swing.JPanel
+import javax.swing.JSeparator
 
 /**
  * Supports creating and managing a [JPanel] for the Settings Dialog.
  */
 class AppSettingsComponent {
-    private val myMainPanel: JPanel
-    private val imagesPathFromModuleText: JBTextField = JBTextField()
-
-    init {
-        myMainPanel = FormBuilder.createFormBuilder()
-            .addLabeledComponent(JBLabel("Enter images directory path from module: "), imagesPathFromModuleText, 1, false)
-            .addComponentFillVertically(JPanel(), 0)
-            .getPanel()
-    }
+  private val myMainPanel: JPanel
+  private val imagesPathFromModuleText: JBTextField = JBTextField()
+
+  private val descriptionText = """
+    <html>To enable the display of Roborazzi tasks, please enable<br>
+    <b>Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)</b> in the settings.</html>
```

Comment:
Could you please clarify the location of the setting? This approach can also be applied to markdown documentation.

```suggestion
    <b>Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)</b> in the Settings | Experimental | Gradle.</html>
```